2011-06-04  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/sv_SE (LC_TIME): Fix d_t_fmt and t_fmt.

diff --git a/localedata/locales/sv_SE b/localedata/locales/sv_SE
index f558e24..2dbc853 100644
--- a/localedata/locales/sv_SE
+++ b/localedata/locales/sv_SE
@@ -169,9 +169,9 @@ mon       "<U006A><U0061><U006E><U0075><U0061><U0072><U0069>";/
           "<U006F><U006B><U0074><U006F><U0062><U0065><U0072>";/
           "<U006E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
           "<U0064><U0065><U0063><U0065><U006D><U0062><U0065><U0072>"
-d_t_fmt   "<U0025><U0061><U0020><U0025><U0065><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0048><U002E><U0025><U004D><U002E><U0025><U0053>"
+d_t_fmt   "<U0025><U0061><U0020><U0025><U0065><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
 d_fmt     "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
-t_fmt     "<U0025><U0048><U002E><U0025><U004D><U002E><U0025><U0053>"
+t_fmt     "<U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053>"
 am_pm     "";""
 t_fmt_ampm  ""
 date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/

